app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+def create_report(data_file_name: str, report_file_name: str) -> None:
+    supply_total = 0
+    buy_total = 0
+
+    data_file = open(data_file_name, "r")
+    for line in data_file:
+        line = line.strip()
+        operation, amount = line.split(",")
+        amount = int(amount)
+
+        if operation == "supply":
+            supply_total += amount
+        elif operation == "buy":
+            buy_total += amount
+    result = supply_total - buy_total
+
+    data_file.close()
+
+    report_file = open(report_file_name, "w")
+    report_file.write("supply," + str(supply_total) + "\n")
+    report_file.write("buy," + str(buy_total) + "\n")
+    report_file.write("result," + str(result) + "\n")
```

Comment:
The reason why I wrote in this format is that flake8 has such an error
![image](https://github.com/user-attachments/assets/a8b270d4-13f0-4503-9f87-dcc306b788d1)